sandbox.go

```diff
@@ -406,6 +407,7 @@ func (sb *sandbox) setupDNS() error {
 	}
 	dnsList := resolvconf.GetNameservers(resolvConf)
 	dnsSearchList := resolvconf.GetSearchDomains(resolvConf)
+	dnsOptionsList := resolvconf.GetOptions(resolvConf)
 
 	if len(sb.config.dnsList) > 0 || len(sb.config.dnsSearchList) > 0 {
```

Comment:
Would it make sense to have dnsOptions specified but no dnsList and no dnsSearchList, so that you would only add the dnsOptions on top of the host's resolv.conf ?

If yes, then the if condition must be changed.